pkg/workloads/console/controller.go

```diff
@@ -898,7 +908,7 @@ func getAuditLogger(logger kitlog.Logger, c *workloadsv1alpha1.Console, statusCt
 	}
 
 	if statusCtx.Authorisation != nil {
-		var subjectNames []string
+		subjectNames := []string{}
```

Comment:
This is just so that in the log event we get `console_authorisers=[]` rather than `console_authorisers=null`